app/templates/components/pagination-pager.hbs

```diff
@@ -4,7 +4,7 @@
   </li>
 
   {{#each pages as |page|}}
-    {{page-item page=page selected=current seperator=seperator urlTemplate=urlTemplate pageSet='pageChanged'}}
+    {{page-item page=page selected=this.current seperator=seperator urlTemplate=urlTemplate pageSet='pageChanged'}}
```

Comment:
[This JSBin](http://jsbin.com/hudaxuxaxe/edit?html,js,output) suggests that the `this.` shouldn't be necessary, but I couldn't get this to work without it. I still can't figure out why.